jccli/config.py

```diff
@@ -0,0 +1,35 @@
+import os
+import configparser
+from pathlib import Path
+
+
+DEFAULT_SECTION = 'DEFAULT'
+CONFIG_FILE_PATH = str(Path.home().joinpath('.jccli.ini'))
+CONFIG_DEFAULTS = {'key': ''}
+
+
+def make_default_config():
+    """Create a config file filled with default values. Only call this if no config file exists, or to overwrite it if
```

Comment:
`..or to overwrite it if it does`?  what does that mean?

I would suggest you don't create one or overwrite or do anything fancy because it will be a burden to maintain especially when you start supporting and deprecating key/value pairs in future versions.  Just document what a config file looks like and tell the user to create it themselves.

Reply:
The goal is just to make a skeleton config file the first time the script is run. I deleted the confusing clause.

Does this make sense?